climada/hazard/test/test_landslide.py

```diff
@@ -19,132 +19,65 @@
-        empty_LS.set_raster(
-            [DATA_DIR.joinpath('test_global_landslide_nowcast_20190501.tif')],
-            window=Window(window_array[0], window_array[1], window_array[3], window_array[2]))
-        empty_LS._intensity_cat_to_prob(max_prob=0.0001)
-        self.assertTrue(max(empty_LS.intensity_cat.data) == 2)
-        self.assertTrue(min(empty_LS.intensity_cat.data) == 1)
-        self.assertTrue(max(empty_LS.intensity.data) == 0.0001)
-        self.assertTrue(min(empty_LS.intensity.data) == 0)
-
-    def test_intensity_prob_to_binom(self):
-        empty_LS = Landslide()
-        window_array = empty_LS._get_window_from_coords(
-            path_sourcefile=DATA_DIR.joinpath('test_global_landslide_nowcast_20190501.tif'),
-            bbox=[47, 23, 46, 22])
-        empty_LS.set_raster(
-            [DATA_DIR.joinpath('test_global_landslide_nowcast_20190501.tif')],
-            window=Window(window_array[0], window_array[1], window_array[3], window_array[2]))
-        empty_LS._intensity_cat_to_prob(max_prob=0.0001)
-        empty_LS._intensity_prob_to_binom(100)
-        self.assertTrue(max(empty_LS.intensity_prob.data) == 0.0001)
-        self.assertTrue(min(empty_LS.intensity_prob.data) == 0)
-        self.assertTrue(max(empty_LS.intensity.data) == 1)
-        self.assertTrue(min(empty_LS.intensity.data) == 0)
-
-    def test_intensity_binom_to_range(self):
-        empty_LS = Landslide()
-        window_array = empty_LS._get_window_from_coords(
-            path_sourcefile=DATA_DIR.joinpath('test_global_landslide_nowcast_20190501.tif'),
-            bbox=[47, 23, 46, 22])
-        empty_LS.set_raster(
-            [DATA_DIR.joinpath('test_global_landslide_nowcast_20190501.tif')],
-            window=Window(window_array[0], window_array[1], window_array[3], window_array[2]))
-        empty_LS._intensity_cat_to_prob(max_prob=0.0001)
-        empty_LS._intensity_prob_to_binom(100)
-        empty_LS.check()
-        empty_LS.centroids.set_meta_to_lat_lon()
-        empty_LS.centroids.set_geometry_points()
-        empty_LS._intensity_binom_to_range(max_dist=1000)
-        self.assertTrue(
-            len(empty_LS.intensity.data[(empty_LS.intensity.data > 0)
-                                        & (empty_LS.intensity.data < 1)]) > 0)
-
-    def test_get_hist_events(self):
-        empty_LS = Landslide()
-        bbox = [48, 23, 40, 20]
-        COOLR_path = DATA_DIR.joinpath('nasa_global_landslide_catalog_point.shp')
-        LS_catalogue_part = empty_LS._get_hist_events(bbox, COOLR_path)
-        self.assertTrue(max(LS_catalogue_part.latitude) <= bbox[0])
-        self.assertTrue(min(LS_catalogue_part.latitude) >= bbox[2])
-        self.assertTrue(max(LS_catalogue_part.longitude) <= bbox[1])
-        self.assertTrue(min(LS_catalogue_part.longitude) >= bbox[3])
-
-
+LS_HIST_FILE = DATA_DIR / 'test_ls_hist.shp'
+LS_PROB_FILE = DATA_DIR / 'test_ls_prob.tif'
+
+class TestLandslideModule(unittest.TestCase):     
+
```

Comment:
Tests for `mapping_point2grid` and `mapping_grid2flattened` are missing.

Reply:
I added them now